fake_spectra/rate_network.py

```diff
@@ -2,50 +2,19 @@
 """A rate network for neutral hydrogen following
 Katz, Weinberg & Hernquist 1996, eq. 28-32."""
 import numpy as np
-import os.path
 import scipy.interpolate as interp
 import scipy.optimize
 
+
+
 class RateNetwork(object):
     """A rate network for neutral hydrogen following
```

Comment:
hmm, we probably want to keep the new comments. There is git magic involving
git reset HEAD\^\^ #Drop the commits but don't change the files
git checkout -p $file #start an ui for selectively dropping changes
git commit -a -m "$message"

to drop individual changes. I can do it if you like though